sos/plugins/lxd.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2016 Jorge Niedbalski <niedbalski@ubuntu.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, UbuntuPlugin
+
+
+class LXD(Plugin, UbuntuPlugin):
+    """LXD is a containers hypervisor.
+    """
+    plugin_name = 'lxd'
+
+    def setup(self):
+        self.add_copy_spec([
+            "/var/log/lxd",
+            "/var/lib/lxd/lxd.db",
+            "/etc/default/lxd-bridge",
+        ])
+
+        # List of containers available on the machine
+        self.add_cmd_output([
+            "lxc list",
+            "lxc profile list",
+            "lxc image list",
+        ])
+
+        self.add_cmd_output([
+            "find /var/lib/lxd -maxdepth 2 -type d",
+        ], suggest_filename='var-lxd-dirs.txt')
```

Comment:
Ah, OK, I missed the `maxdepth` - in that case it would be preferred to use the `-ls` switch to `find` in order to have the output format as similar as possible to other plugins that collect this kind of information (the depth restriction is fine - it's a shame that `ls` cannot do this itself).